src/include/main/connection.h

```diff
@@ -136,6 +136,18 @@ class Connection {
      */
     KUZU_API void interrupt();
 
+    /**
+     * @brief sets the query timeout value of the current connection. This will also enables the
+     * query timeout of the current connection.
+     */
+    KUZU_API void setQueryTimeOut(uint64_t timeoutInMS);
+
+    /**
+     * @brief sets the query timeout enabled of the current connection. Query timeout is disabled by
+     * default.
+     */
+    KUZU_API void setQueryTimeOutEnabled(bool isEnabled);
```

Comment:
I want to discuss these two newly added public functions with you and @andyfengHKU a bit more. I would suggest have two functions:
```
void enableQueryTimeout(uint64_t timeoutInMS);
void disableQueryTimeout();
```
Compared to the current two functions, the benefit is that when a user wants to set a timeout for a query, just one function needs to be called, instead of two.

Reply:
Actually, as we discussed, I think we should just follow Postgres on this.